playbook/app/pb_kits/playbook/pb_text_input/_text_input.jsx

```diff
@@ -85,7 +89,7 @@ const TextInput = (props: TextInputProps, ref: React.ElementRef<"input">) => {
   )
   const textInput = (
     <input
-        {...props}
+        {...domSafeProps}
```

Comment:
Doesn't this still needs to be `{...props}`, @gmfvpereira ... because you are spreading `domSafeProps` above on line 41, but the name on line 42 is still `props`?

Reply:
Otherwise, you can just name line 42 to be domSafeProps... and keep this one the same.

Reply:
> Doesn't this still needs to be `{...props}`, @gmfvpereira ... because you are spreading `domSafeProps` above on line 41, but the name on line 42 is still `props`?

When we spread we are doing a copy of the values. After that line `props.addOn` is still there. So if I use `...props` we'll keep forwarding `addOn` to the DOM.

>  Or.. you can just name line 42 to be domSafeProps... and keep this one the same.

I'm not sure I got your suggestion, like this?
```
  const {
    aria = {},
    className,
    data = {},
    dark = false,
    disabled,
    error,
    id,
    inline = false,
    name,
    label,
    onChange = () => {},
    placeholder,
    required,
    type = 'text',
    value = '',
    children = null,
    addOn = { icon: null, alignment: 'right', border: true },
    ...domSafeProps,
  } = props
```
If so, I consider that too, but that means that we would stop forwarding to the DOM all the other properties, like `id`, `name`, and so on and we don't want that.

Reply:
Ah thanks @gmfvpereira ! Gabriel synced with me over zoom on this and got me squared away! Thanks man.